pages/glossary/essential-text-change.md

```diff
@@ -0,0 +1,15 @@
+---
+title: Essential Text Change
+key: essential-text-change
+unambiguous: true
+objective: true
+input_aspects:
+  - Accessibility tree
+  - CSS styling
+  - DOM tree
+---
+
+An _essential text change_ occurs when the text content of an HTML element is required to convey accurate information to the user. An _essential text change_ may appear in any of the following forms:
+
+1. Providing context - Text is changed to provide users with additional information required to understand the current state of the page. For example, while searching for results, text may change to convey the page is *loading* to provide context to the user.
+2. Live updates of important information - The content automatically updates to show the most recent information available from a data source and should not be ignored by the user. 
```

Comment:
This feels very awkward, namely specifying that the information must be important in order for this to qualify. I felt the need to do this since I wouldn't qualify the examples given in Understand 2.2.2 like weather or stock ticker information as important enough to be exempt from having controls. *However*, there could be information such as safety alerts/updates that I would consider to be exempt.

Reply:
Yeah, this one's difficult. Who decides what should and what should not be ignored? I think we should maybe just try to give a list of the types of things that shouldn't be ignored? Maybe, financial data, information on the status of a user session, that kind of thing.